internal/venice-common/src/test/java/com/linkedin/venice/controllerapi/TestStoppableNodeStatusResponse.java

```diff
@@ -0,0 +1,70 @@
+package com.linkedin.venice.controllerapi;
+
+import static org.testng.Assert.assertEquals;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.linkedin.venice.utils.ObjectMapperFactory;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.testng.annotations.Test;
+
+
+public class TestStoppableNodeStatusResponse {
```

Comment:
Do we need this test class ? The code path being tested is executed in other places

Reply:
This is to test the JSON serde of `StoppableNodeStatusResponse` class. The coverage check fails without this as `StoppableNodeStatusResponse` is in `venice-common`.

Let me modify the class a bit to remove the inner `StoppableNodeStatusResponseForTest` class